common/browser.go

```diff
@@ -28,8 +26,11 @@ const (
 
 // Browser stores a Browser context.
 type Browser struct {
-	ctx      context.Context
-	cancelFn context.CancelFunc
+	initContext  context.Context
+	initCancelFn context.CancelFunc
```

Comment:
Can you add a comment to clarify what "init" means in this context? 🙇

Reply:
Sure, added in 41ad3e0844a83865b13ac1009cefa118b20ce775